docs/7-vector-search/5-aws-sagemaker.mdx

```diff
@@ -0,0 +1,109 @@
+
+## Let's set up a playground for you to try it out!
+
+On the main page of SageMaker you'll find the option to `Set up for single user` or `Set up for organization`. These will set up a domain and a quick start user.
+
+<Screenshot src="img/screenshots/7-vector-search/5-aws-sagemaker/1-main-page.png" url="https://aws.amazon.com/pm/sagemaker/" alt="AWS SageMaker" />
+<Screenshot src="img/screenshots/7-vector-search/5-aws-sagemaker/2-setup-started.png" url="https://aws.amazon.com/pm/sagemaker/" alt="AWS SageMaker" />
+<Screenshot src="img/screenshots/7-vector-search/5-aws-sagemaker/3-preparing-domain.png" url="https://aws.amazon.com/pm/sagemaker/" alt="AWS SageMaker" />
+
+The initial setup only has to be done once, but it might take several minutes. When finished, SageMaker will notify you about the new domain being ready.
+
+[Amazon SageMaker Domain](https://docs.aws.amazon.com/sagemaker/latest/dg/sm-domain.html) supports SageMaker machine learning (ML) environments. A SageMaker Domain is composed of the following entities:
+- `Domain`: An Amazon SageMaker Domain consists of an associated Amazon Elastic File System (Amazon EFS) volume; a list of authorized users; and a variety of security, application, policy, and Amazon Virtual Private Cloud (Amazon VPC) configurations. Users within a Domain can share notebook files and other artifacts with each other. An account can have multiple Domains. For more information about multiple Domains, see Multiple Domains Overview.
+- `UserProfile`: A user profile represents a single user within a Domain. It is the main way to reference a user for the purposes of sharing, reporting, and other user-oriented features. This entity is created when a user onboards to the Amazon SageMaker Domain. For more information about user profiles, see Domain User Profiles.
+- `shared space`: A shared space consists of a shared JupyterServer application and shared directory. All users within the Domain have access to the shared space. All user profiles in a Domain have access to all shared spaces in the Domain. For more information about shared spaces, see Collaborate with shared spaces.
+- `App`: An app represents an application that supports the reading and execution experience of the user’s notebooks, terminals, and consoles. The type of app can be JupyterServer, KernelGateway, RStudioServerPro, or RSession. A user may have multiple apps active simultaneously.
+
+<Screenshot src="img/screenshots/7-vector-search/5-aws-sagemaker/4-domain-ready.png" url="https://aws.amazon.com/pm/sagemaker/" alt="AWS SageMaker" />
+
+Now that the domain is set up, you want to open one of the built-in apps to actually work on your environment.
+
+Amazon SageMaker supports the following [machine learning environments](https://docs.aws.amazon.com/sagemaker/latest/dg/machine-learning-environments.html).
```

Comment:
I personally would skip this, and focus on the "how to implement" part, but up to you.